doc/source/tutorial/stats/continuous_gamma_gompertz.rst

```diff
@@ -0,0 +1,15 @@
+
+.. _continuous-gamma_gompertz:
+
+Gamma/Gompertz Distribution
+========================================
+
+For :math:`x\geq0`, :math:`c>0`, and :math:`\beta>0` . 
+
+.. math::
+
+    f\left(x;c,\beta \right) = \frac{c {e^x} {{\beta }^c} }{ {\left( \beta - 1 + {e ^ x} \right) }^ {c+1} }
```

Comment:
This could use some additional information.  Not just `f`', but also `F`, `G`, `S`, 'Z`, the mean and variance.

Reply:
Unforunately, the mean and variance of this distribution are very complex. Could we follow the convention of other *.rst files and just provide one math formula?